gamemodes/clockwork/framework/items/bases/sh_clothes_base.lua

```diff
@@ -451,6 +451,13 @@ local ITEM = item.New(nil, true);
 					if !player:IsNoClipping() and (!player.GetCharmEquipped or !player:GetCharmEquipped("urn_silence")) then
 						local useSound = self("useSound");
 						
+						if self.uniqueID == "darklander_immortal_armor" then	
+							player:EmitSound("piggysqueals/transformation/"..math.random(1, 2)..".mp3");
+							player:Disorient(8)
+							timer.Create( "Squeal", 5.5, 1, function() netstream.Start(player, "Stunned", 3); end )
```

Comment:
```suggestion
							timer.Create( "Squeal", 5.5, 1, function() if IsValid(player) then netstream.Start(player, "Stunned", 3); end end)
```
nitpick, this can be NULL if the player disconnects